docs/source/principles.md

```diff
@@ -47,22 +47,23 @@ TREs holding public data should build and maintain the trust of data subjects an
 
 ### Rationale
 
-Maintaining trust in TREs to hold and use data is essential to prevent backlash or resistance to the use of data.
-This can include maintaining the trust of members of the public who are impacted by research, as well as the trust of commercial data providers.
+To ensure continued public support for the use of data and to alleviate possible concerns, it is vital to maintain public trust in the ways TREs hold and use data.
+This can include maintaining the trust of members of the public whose data is held, those who are impacted by research, as well as the trust of commercial data providers
```

Comment:
```suggestion
This could include maintaining the trust of members of the public whose data is held, those who are impacted by research conducted using TREs, and the trust of commercial data providers.
```